server/pulp/server/webservices/views/serializers/dispatch.py

```diff
@@ -47,3 +48,21 @@ def task_status(task):
     task_dict['id'] = str(task['id'])
 
     return task_dict
+
+
+def group_task_summary(tasks):
```

Comment:
I don't think this is a serializer. It creates a new document based on a stream of input. A serialization operation should be something that can be undone. If you can't deserialize and get the original data, then you probably didn't do a serialization operation. I would put this with the rest of the task view, or make a controller method.